common/metrics/metric_defs.go

```diff
@@ -469,6 +469,8 @@ const (
 	MatchingClientInvalidateTaskQueueUserDataScope = "MatchingClientInvalidateTaskQueueUserData"
 	// MatchingClientGetTaskQueueUserDataScope tracks RPC calls to matching service
 	MatchingClientGetTaskQueueUserDataScope = "MatchingClientGetTaskQueueUserData"
+	// MatchingClientApplyTaskQueueUserDataReplicationEventScope tracks RPC calls to matching service
+	MatchingClientApplyTaskQueueUserDataReplicationEventScope = "MatchingClientApplyTaskQueueUserDataReplicationEventScope"
```

Comment:
I realize I left the Scope suffix in the string here and in the other scope, will fix in the next PR.